[WIP] TELCODOCS-1707: Lifecycle Agent GA (#74025)
openshift/openshift-docs: edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.adoc

:_mod-docs-content-type: ASSEMBLY
[id="cnf-image-based-upgrade-for-sno"]
= Performing an image-based upgrade for {sno} clusters
include::_attributes/common-attributes.adoc[]
:context: cnf-image-based-upgrade

toc::[]

> The {lcao} allows you to perform an image-based upgrade on your {sno} clusters manually or through {rh-rhacm}.

Reviewer: ```suggestion
You can use the {lcao} to do a manual image-based upgrade of a {sno} cluster.
```
